gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/setWriter.ts

```diff
@@ -1,15 +1,22 @@
 import { Result } from '@chainlink/gauntlet-core'
-import { logger, BN } from '@chainlink/gauntlet-core/dist/utils'
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
 import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
 import { PublicKey } from '@solana/web3.js'
+import { utils } from '@project-serum/anchor'
 import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+import { getRDD } from '../../../lib/rdd'
+
+type Input = {
```

Comment:
Changed a bit the command interface to be more understandable. User should provide the ocr2 state address as flag. If RDD is provided, transmissions and store should be there. 
`storeAuthority` is calculated on the fly